recipe/run_test.py

```diff
@@ -121,12 +122,12 @@
     if sys.platform == 'win32':
         TCLTK_VER = '8.5'
     else:
-        TCLTK_VER = '8.6'
+        TCLTK_VER = os.getenv("tk")
```

Comment:
This will fail because the environment variable reports the full version, 8.6.8, but `_tkinter.TK_VERSION` only reports major.minor, 8.6.  `os.getenv("tk")[:3]` could be used or `TCLTK_VER.startswith(_tkinter.TCL_VERSION)`

Reply:
Hmmm...  are you sure? the env var is the variant value.  This formulation is what we have in 3.6 and 3.7, and it seems OK.

Reply:
Oops, you are correct, this work fine.  I ran a build locally from the wrong directory and was not picking up the variant information from `conda_build_config.yaml`.

Reply:
👍